src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -935,7 +938,7 @@ export class ResultsViewPageStore {
                         }
                     }));*/
                     ret.push(profile);
-                } else {
+                } else if (profile.molecularAlterationType !== AlterationTypeConstants.STRUCTURAL_VARIANT) { //All studies except structural variants
```

Comment:
we should probably refactor this whole if statement as switch.  would be more readable.  not too much work.